presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadataFactory.java

```diff
@@ -136,6 +139,7 @@ public HiveMetadataFactory(
         }
 
         renameExecution = new BoundedExecutor(executorService, maxConcurrentFileRenames);
+        metastoreDropExecutor = new BoundedExecutor(executorService, maxConcurrentMetastoreDrops);
```

Comment:
just call it `dropExecutor` for consistency with `renameExecution`

Reply:
done